packages/variable-editor/src/components/variables/master/AddVariableDialog.tsx

```diff
@@ -73,7 +79,7 @@ export const AddVariableDialog = ({ table, variables, setVariables, setSelectedV
             <Input />
```

Comment:
Maybe not part of this pr, but a very good gif. We should not allow an empty name, but show a message if someone tries to add a variable with no name

Reply:
Going to add validation with another PR :+1: